libs/designer/src/lib/ui/panel/templatePanel/templatePanel.tsx

```diff
@@ -6,15 +6,23 @@ import { closePanel } from '../../../core/state/templates/panelSlice';
 import { CreateWorkflowPanel } from './createWorkflowPanel/createWorkflowPanel';
 import { QuickViewPanel } from './quickViewPanel/quickViewPanel';
 
-export const TemplatePanel = () => {
+export const TemplatePanel = (
+  // { onCreateClick }: { onCreateClick: () => Promise<void> }
+) => {
   const dispatch = useDispatch<AppDispatch>();
   const { isOpen, currentPanelView } = useSelector((state: RootState) => state.panel);
 
   const dismissPanel = useCallback(() => dispatch(closePanel()), [dispatch]);
 
   return (
     <Panel isLightDismiss type={PanelType.medium} isOpen={isOpen} onDismiss={dismissPanel} hasCloseButton={true}>
-      {currentPanelView === 'createWorkflow' ? <CreateWorkflowPanel /> : currentPanelView === 'quickView' ? <QuickViewPanel /> : undefined}
+      {currentPanelView === 'createWorkflow' ? (
+        <CreateWorkflowPanel
+        // onCreateClick={onCreateClick}
+        />
+      ) : currentPanelView === 'quickView' ? (
+        <QuickViewPanel />
+      ) : undefined}
```

Comment:
this should return null right? we either return the component or null, why undefined?